### ISSUE TITLE
Websocket needs an absolute URL

### ISSUE THREAD

**Issue by ab-pm** [prisma/graphql-playground#1076]
#### This issue pertains to the following package(s):

- [ ] GraphQL Playground - Electron App
- [x] GraphQL Playground HTML
- [x] GraphQL Playground
- [ ] GraphQL Playground Express Middleware
- [ ] GraphQL Playground Hapi Middleware
- [ ] GraphQL Playground Koa Middleware
- [ ] GraphQL Playground Lambda Middleware

(I guess?)

#### What version of are you experiencing the issue(s) on?

The latest https://cdn.jsdelivr.net/npm/graphql-playground-react/build/static/js/middleware.js.
We're on 
* graphql-playground-html@1.6.12
* graphql-playground-middleware-express@1.7.12

#### What's the issue

The issue is two-fold:
* When passing only a path, not a fully-qualified URL, as the endpoint, then subscriptions don't work because the WebSocket fails to be constructed:
  `DOMException: Failed to construct 'WebSocket': The URL '/graphql' is invalid.`
  In short, it breaks with
  ```js
  expressPlayground({
  	endpoint: '/graphql',
  })
  ```
  but works with
  ```js
  expressPlayground({
  	endpoint: `${BACKEND_URL}/graphql`,
  })
  ```
  It should take the current location into account, just like normal queries do.
* When changing the URL in the endpoint input field, it doesn't affect subscriptions but only normal queries.

**Comment by iamnotaturtle** [comment]
Thanks! Providing the full URL fixed this issue:
```
Failed to construct 'WebSocket': The URL '/graphql' is invalid.
```
